### DISCUSSION TITLE
有没有基于go-cqhttp的linuxQQ前端，或者类似第三方LinuxQQ客户端

### DISCUSSION THREAD

**Discussion by xiazhuozhao** [Mrs4s/go-cqhttp#1662] [General]
实在是受不了linuxQQ了

**Comment by Mrs4s** [comment]
比较刑 没人敢做

**Comment by Ink-33** [comment]
我觉得可以期待一下也许有可能出来的用Electron重构的macos/linux qq

**Reply by xiazhuozhao** [reply]
同，今天看到了Electron重构的macQQ，感觉离linuxqq的曙光不远了啊

**Comment by clacknya** [comment]
蹲一个

**Comment by ctaoist** [comment]
发现一个这个，https://github.com/Stapxs/Stapxs-QQ-Lite
不过对 go-cqhttp 支持不是太完整，主要是缺一个获取一个历史消息的接口，我正在这个接口，不过只会 mongodb，只能有限支持mongodb了

已经写完了： #1742

**Comment by ctaoist** [comment]
还有这个 https://github.com/Icalingua-plus-plus/Icalingua-plus-plus
